docker-compose.migrate.yaml

```diff
@@ -47,4 +45,4 @@ services:
       - postgres_data:/var/lib/postgresql/data
 
 volumes:
-  postgres_data: {}
+  postgres_data:
```

Comment:
I noticed that this was the only file that had this change out of all the other peer reviews - was this a mistake?

Reply:
No, I just spotted that this was unnecessary when I got to this one.